src/main/java/bio/terra/app/logging/LoggerInterceptor.java

```diff
@@ -14,40 +16,75 @@
 
 @Component
 public class LoggerInterceptor implements HandlerInterceptor {
-  private static Logger logger = LoggerFactory.getLogger(LoggerInterceptor.class);
+  private static final Logger logger = LoggerFactory.getLogger(LoggerInterceptor.class);
+  // Constants for requests coming in that aren't authenticated
+  private static final String UNAUTHED_USER_ID = "N/A";
+  private static final String UNAUTHED_EMAIL = "N/A";
+  private static final String UNAUTHED_INSTITUTE = "N/A";
+  // Don't log requests for URLs that end with any of   the following paths
+  private static final Set<String> LOG_EXCLUDE_LIST = Set.of("/status");
+
+  private static final String REQUEST_START_ATTRIBUTE = "x-request-start";
+  private static final Long NOT_FOUND_DURATION = -1L;
```

Comment:
For values that can't be `null`, I think it's clearer to use an unboxed type.
```suggestion
  private static final long NOT_FOUND_DURATION = -1;
```

Reply:
Ah fair point